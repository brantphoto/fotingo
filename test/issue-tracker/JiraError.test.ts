import { JiraErrorImpl } from 'src/issue-tracker/jira/JiraError';
import { describe, expect, it } from 'vitest';

describe('JiraError', () => {
  it('should store the message and the code', () => {
    const error = new JiraErrorImpl('Test message', 'ERR_CODE');
    expect(error.message).toMatchSnapshot();
    expect(error.code).toMatchSnapshot();
  });
});
