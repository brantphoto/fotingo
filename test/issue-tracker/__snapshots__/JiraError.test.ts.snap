// Vitest Snapshot v1

exports[`JiraError > should store the message and the code 1`] = `"Test message"`;

exports[`JiraError > should store the message and the code 2`] = `"ERR_CODE"`;
